src/components/PlayerInfo.vue

```diff
@@ -8,15 +8,28 @@
       <span v-if="round<6">Income: {{player.income.replace(/,/g, ', ')}}<br/></span>
       Range: {{data.range}}, Terraforming cost: {{3 - data.terraformCostDiscount}}o<br/>
       <span v-if="faction === 'Ivits'">Fed value: {{player.fedValue }}, No fed value: {{player.structureValue - player.fedValue }} <br/></span> 
-
-      <span style="white-space: nowrap; line-height: 1em">
+ 
+        <span style="white-space: nowrap; line-height: 1em">
         Steps: 
         <span v-for="i in [0, 1, 2, 3]" :key="i" :class="{'ml-2': i > 0}">
-          <i v-for="planet in planetsWithSteps(i)" :class="['planet', planet]" :key="planet" /> {{i}}
+          <span v-for="planet in planetsWithSteps(i)"  :key="planet"  >
+            <svg width="15" height="20" viewbox="0 0 15 15" >
+              <circle :cx="8" :cy="8" :r="6"  :class="['player-token', 'planet-fill', planet]" />
+            </svg>
+          </span> 
+          <span>{{i}}</span>
         </span>
       </span><br/>
-      <span style="line-height: 1em" v-if="hasPlanets">Colonized: <span v-for="(count, planet, index) in player.ownedPlanetsCount" :key="planet" :class="{'ml-2': index > 0}"><i :class="['planet', planet]" :key="planet" /> {{count}} </span></span>
+      <span style="line-height: 1em" v-if="hasPlanets">Colonized: 
+        <span v-for="(count, planet, index) in player.ownedPlanetsCount"  :key="planet"  :class="'ml-2'">
```

Comment:
No need to use `:class` as there's no 'code', you can just do `class="ml-2"`. To make it consistent with the steps, use `mr-2` instead so there's no margin after "Colonized: ".

Reply:
ok